images/nginx/rootfs/build.sh

```diff
@@ -145,7 +145,7 @@ get_src 30affaf0f3a84193f7127cc0135da91773ce45d902414082273dae78914f73df \
         "https://github.com/rnburn/zipkin-cpp-opentracing/archive/v$ZIPKIN_CPP_VERSION.tar.gz"
 
 get_src 5c8d25e68fb852f61489b669aebb7bd8ca8c88ebb5e5f969212fcceff3ee2d0b \
```

Comment:
Please update the sha256 of the new tar.gz file

Reply:
Thanks, done.